### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ivy** [sourcegraph/go-selenium#40]
Greetings! It looks like this project is no longer being developed while upstream ([`tebeka/selenium`](https://github.com/tebeka/selenium)) is looking very active.

Are there any plans to develop this package further or is it dead? :confused:

**Comment by sqs** [comment]
Correct, this one is no longer being developed. It looks like the upstream has made a lot of progress recently. I'll update the readme to point people to the upstream.

**Comment by pwaller** [comment]
Can someone clarify how `tebeka/selenium` is "upstream"? (I don't see that it is upstream...) It doesn't look better or worse than this project from what I see.

**Comment by pwaller** [comment]
Oh, I just spotted the [note in the readme](https://github.com/sourcegraph/go-selenium#contributors) that says this code was based on tebeka originally. And also it looks like that code now has a lot of recent activity on it.
